src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1309,7 +1309,10 @@ public function registerForAutoconfiguration(string $interface)
     /**
      * Registers an attribute that will be used for autoconfiguring annotated classes.
      *
-     * The configurator will receive a ChildDefinition instance, an instance of the attribute and the corresponding \ReflectionClass, in that order.
+     * If an attribute can be used on multiple targets, a union type should be used for
+     * the \Reflector parameter. Oterwise specify one \Reflector type.
+     *
+     * @param callable(ChildDefinition, class-string $attribute, \ReflectionClass|\ReflectionMethod|\ReflectionProperty|\ReflectionParameter):void $configurator
```

Comment:
`class-string $attribute` is not correct. $attributeClass is a class-string

Reply:
this should be `object $attribute`

Reply:
Fixed. Added `@param class-string $attributeClass` as well.